docs/api.rst

```diff
@@ -2521,7 +2542,8 @@ of :class:`enum.Enum`.
         A member was kicked.
 
         When this is the action, the type of :attr:`~AuditLogEntry.target` is
-        the :class:`User` who got kicked.
+        the :class:`User` who got kicked. If the user is not found then it is
+        a :class:`Object` with the user's ID.
```

Comment:
A similar change is needed for `.message_delete`, `.message_pin`, `.message_unpin`, `.automod_block_message`, `.automod_send_alert_message` and `.automod_timeout`, all of which may have an `Object` target

Reply:
b8b241adc9923bab679daca945fb28bc22a18dc2